### PR TITLE
remove buggy code

### PR DESCRIPTION
**Pull request by studentmain**
## Please follow the guide below

- You will be asked some questions, please read them **carefully** and answer honestly
- Put an `x` into all the boxes [ ] relevant to your *pull request* (like that [x])
- Use *Preview* tab to see how your *pull request* will actually look like

- [x] [Searched](https://github.com/shadowsocks/shadowsocks-windows/search?q=is%3Apr&type=Issues) for similar pull requests
- [ ] Compiled the code with Visual Studio
- [ ] Require translation update
- [ ] Require document update (readme.md, wikipage, etc)

### What is the purpose of your *pull request*?
- [x] Bug fix
- [ ] Improvement
- [ ] New feature

---

### Description of your *pull request* and other information

remove buggy stream cipher and its test

### CODE REVIEW

**Comment by DuckSoft** [comment]
LFGTM

File: shadowsocks-csharp/Encryption/EncryptorFactory.cs [hunk 1]
```diff
@@ -26,7 +26,7 @@ static EncryptorFactory()
             {
                 AEADSodiumEncryptorSupportedCiphers.Remove("aes-256-gcm");
             }
-
+#if I_KNOW_STREAM_CIPHER_IS_UNSAFE
```
Comment (DuckSoft) [inline shadowsocks-csharp/Encryption/EncryptorFactory.cs:29]: I'd rather put: if i am sb

**Review by DuckSoft — APPROVED** [review]

**Comment by database64128** [comment]
All these overused insecure legacy stream ciphers. Let’s drop them once and for all! 🚀

**Comment by eebssk1** [comment]
I'm using v2ray-pluin TLS while setting SS to use rc4-md5. Is it  really necessary to use AEAD on both plugin and SS itself?

**Comment by studentmain** [comment]
@eebssk1 using v2ray (without plugin) is enough

File: shadowsocks-csharp/Encryption/EncryptorFactory.cs (hunk 1 above)
Comment (fernvenue) [reply shadowsocks-csharp/Encryption/EncryptorFactory.cs:29]: > 

So maybe you can tell me why we have to use aead cipher when we already have TLS by [v2ray-plugin](https://github.com/shadowsocks/v2ray-plugin). Or shadowsocks-windows is going to drop support for plugins at a later date?

**Pull request merged**